### DISCUSSION TITLE
Help Needed with lumi_lm-s4zw/s*zw Switch in z-wave-js‏‏

### DISCUSSION THREAD

**Discussion by asaf2298** [zwave-js/zwave-js#5653] [Q&A]
Hii everyone, 

I will start with that I am new to Z Wave Js and I don't know if this is the right place so forgive me if I'm wrong.

I am reaching out here for help with a persistent issue I've been experiencing for over a year - managing the lumi_lm-s4zw (2 on/off light switches and 2 up/down blinds switches) switch with z-wave-js. 

Upon setting the switch, I've noticed that it displays three regular buttons instead of the expected two, along with an additional up/down buttons for the ruler. Unfortunately, pressing these buttons in z-wave-js leads to random behavior and a dead node state. 

I've compiled some relevant information that I'm attaching to this message. 

Thank you for your time, and I hope that you can assist me in resolving this issue.

https://devices.zwave-js.io/?jumpTo=0x0293:0x0003:0x0024:0.0
https://drive.google.com/drive/folders/18k_JEDgfYBOpRch5ThUz6dLzqfayZTGK?usp=sharing

**Comment by AlCalzone** [comment]
Please make a [driver log](https://zwave-js.github.io/zwave-js-ui/#/troubleshooting/generating-logs?id=driver-logs), loglevel `debug` and attach it here as a file (drag & drop into the text field).

The driver log should include a re-interview plus this:
> Upon setting the switch, I've noticed that it displays three regular buttons instead of the expected two, along with an additional up/down buttons for the ruler. Unfortunately, pressing these buttons in z-wave-js leads to random behavior and a dead node state.

**Reply by asaf2298** [reply]
Hii,
first of all thank you for the respond and sorry for the long waited answer, I hope you can still find time to replay. 
I did a log of all nodes while playing with problematic one and another log filtered to another problematic switch. hope I did it right. 

[zwave-js-ui-store.zip](https://github.com/zwave-js/node-zwave-js/files/11790328/zwave-js-ui-store.zip)
[Log all.zip](https://github.com/zwave-js/node-zwave-js/files/11790329/Log.all.zip)

You can also see there are 3 buttons for this 2 buttons and 1 roller switch 

![IMG_7893](https://github.com/zwave-js/node-zwave-js/assets/120143624/d1fd3803-eeea-4420-a6b6-cad371d0e5c5)

**Reply by AlCalzone** [reply]
Unfortunately, all I see in that log is that the device is unreachable. That's why it gets marked as dead.

You'll want to work through the troubleshooting guides for this:
- https://zwave-js.github.io/node-zwave-js/#/troubleshooting/connectivity-issues?id=general-troubleshooting
- https://zwave-js.github.io/node-zwave-js/#/troubleshooting/network-health?id=testing-the-connection-strength
